content/pages/getting_started/what_do_i_need.md

```diff
@@ -0,0 +1,96 @@
+
+### Shoestring
+
+Suggestions for those looking to get started as cheaply as possible.
+
+* Hammer: 2-2.5lb (900-1100g) crosspein, rounding, or drilling hammer from your local big box store. Check out our guide to [dressing your hammer][].
+* Forge: [Just a Box of Dirt][jabod]
+* Anvil: [Sledgehammer head][sledgehammer anvil] or 4140/1045 offcut from eBay or your local steelyard or machine shop
+* Tongs: Use long stock or [Ken's Custom Iron tong blanks][].
+
+### Economy
+
+If you've got a little more cash to burn, these options might make your life a little easier.
+
+* Hammer: 2-2.5lb (900-1100g) crosspein, rounding, or drilling hammer from your local big box store. Check out our guide to [dressing your hammer][].
+* Forge: [Mr. Volcano single burner][Mr. Volcano forge]
+* Anvil: [Harbor Freight Doyle cast steel anvil][] or check out our [anvil buying guide][]
+* Tongs: 3/8" V-bit bolt jaw or wolf jaw from [Centaur Forge][], [Blacksmiths Depot][], or [Texas Farrier Supply][].
+
+### Premium
+
+Fine goods for the discerning aspiring blacksmith
+
+* Hammer: 2-2.5lb (900-1100g) crosspein or rounding hammer from one of your fellow artists: [Brent Bailey][]
+* Forge: [Mr. Volcano single or double burner][Mr. Volcano forge] or [Chile forge][]
+* Anvil: [Anvil buying guide][] or buy new from [Texas Farrier Supply][]
+* Tongs: 3/8" V-bit bolt jaw or wolf jaw from [Centaur Forge][], [Blacksmiths Depot][], or [Texas Farrier Supply][]
+
+## Nice-to-Haves: Upgrading your shop
+Forging with just the basics will get frustrating fast. A post vise is an excellent upgrade - unlike conventional bench vises, a post vise is designed to stand up to repeated heavy hammering by transferring the force of the hammer blows through the post into the floor.
+
+For folks interested in bladesmithing, a belt grinder is ideal; the bulk of bladesmithing operations take place at the grinder. If you're on a budget, a [1x30][] belt grinder will get you started, but if you have a bit more to spend, a [2x72][] is a worthwhile investment. However, if a belt grinder isn't an option for you, an angle grinder or even a set of [files][Pferd files], together with a bit of patience, will yield perfectly respectable results.
+
+### Our recommendations:
+
+* Angle grinder
+* Post vise
+* Chop saw
+* Belt grinder [1x30][] | [2x72][]
+* [Welder][]
+
+## Going pro (as in production)
+None of the equipment mentioned in this section is needed in a beginner's shop, but if you're looking to produce at scale, you may want to consider investing in a machine or two.\
+
+Our recommendations:\
+* [TIG welder][]
+* Press
+* Power hammer
+* Heat treat oven
+
+[jabod]: /pages/equipment/jabod
+[Harbor Freight Doyle cast steel anvil]: https://www.harborfreight.com/65-lb-cast-steel-anvil-58924.html
+[Ken's Custom Iron tong blanks]: https://kensironstore.com/collections/quick-tongs
+[Mr. Volcano forge]: https://mrvolcano.com/collections/the-forge
```

Comment:
No longer a reliable option

Reply:
Per our discussion, we need to come to a consensus on what we should be recommending to newbies going forward, and make this change in a separate PR.